examples/gridViewExample/build.gradle

```diff
@@ -25,6 +25,6 @@ tasks.preBuild {
 }
 
 dependencies {
-    compile 'com.google.code.gson:gson:2.3'
+    compile 'com.google.code.gson:gson:1.7.1'
```

Comment:
Slightly annoying though is that Retrofit uses GSON 2.3.1, which could cause a lot of complaints coming our way (even though it is bugs in GSON). I'll dig a little deeper to see if there is a solution that can be used in 2.3.1.